build.gradle

```diff
@@ -562,6 +562,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind
```

Comment:
I forgot this dependency change would happen. You almost definitely should ping the mailing list with this -- dependency changes on the clients lib are pretty significant, especially a widely used one like Jackson that can easily conflict (although I think they are very good about compatibility).

Another option would be to use a separate jar -- still move out of connect, but put it in something like a `kafka-serde-json` jar. Streams code could easily depend on that without depending on all of connect, but it wouldn't require everyone using Kafka clients to pull in Jackson. (In general I think Kafka should be broken into finer-grained jars).